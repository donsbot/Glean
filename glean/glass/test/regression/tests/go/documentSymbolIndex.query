action: documentSymbolIndex
args: {"include_refs":true,"filepath":"leaphash.go","repository":"test"}

