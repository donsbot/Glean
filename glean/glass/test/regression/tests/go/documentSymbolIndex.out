[
    "@generated",
    {
        "revision": "testhash",
        "size": 0,
        "symbols": {},
        "truncated": false
    }
]